delete_file.php

```diff
@@ -3,9 +3,20 @@
 require __DIR__ . '/vendor/froala/wysiwyg-editor-php-sdk/lib/FroalaEditor.php';
 
 try {
-  $response = FroalaEditor_File::delete($_POST['src']);
-  echo stripslashes(json_encode('Success'));
+	$src = $_POST['src'];
+	$upload_folder = "/uploads/";
+
+	// If the file is inside the uploads folder
+	if (substr($src, 0, strlen($upload_folder)) === $upload_folder)
```

Comment:
This doesn't really fix the problem because the path might be something like this: `/uploads/../foo/something_bad_here`. Also, this should be done in the SDK, not in the examples.

Reply:
Why the user would have something bad in the uploads folder?
The SDK don't have any kind of "users" extensions to check permissions if the user can delete the file, so how should this be handled?

Reply:
Inside the SDK when deleting a file, there should be a check to see if the `$upload_folder` actually contains the file indicated by the path. The easiest way would be to look into the `$upload_folder` files recursively and compare paths. Again, this should be done in the SDK itself, not here.